### PR TITLE
Shilpi's Week 5 blog

### PR DESCRIPTION
**Pull request by shilpiprd**

### CODE REVIEW

**Review by skoudoro — CHANGES_REQUESTED** [review]
Hi @shilpiprd,

file is empty ?

**Comment by shilpiprd** [comment]
Sorry, I think this was a confusion. I'll upload here right away.

**Comment by skoudoro** [comment]
Hi @shilpiprd,

- Blog 6th, has been merged so no need to add it to this PR.
- Blog 5th is still empty

**Comment by shilpiprd** [comment]
Hey Serge, it seems on the official blog page, I haven't uploaded Week1 blog and I've renamed Week2 blog as Week1. So should I make a submission now for Week1 blog and the increase 1 on all weeks from there after ? 
Thanks,

**Comment by skoudoro** [comment]
just call it week0 it is fine

**Comment by shilpiprd** [comment]
> 

Well, the thing is that I'm starting from Week0 in my official blog as well. Also, I think Rafael said a few days ago that I haven't submitted 1 blog meaning I'm in the 1 missing blog zone. So this is basically the reason why on my blog I'm posting Week10, but in dipy blog page it's Week11. So, should I make changes in PSF or leave it as it is ?

**Comment by skoudoro** [comment]
You can leave it as it is. 

You will just have to pay attention when you create you final report. You final report should look like this: https://fury.gl/latest/posts/2023/2023-01-24-final-report-praneeth.html

on the last section, you can see link to PSF blog and fury blog. You will have to do the same on PSF blog + DIPY blog

**Comment by shilpiprd** [comment]
Serge actually I just noticed  that in the end I would have to list out all the blogs, both in PSF and DIPY side by side. Problem is that Week5 in PSF and Week5 in DIPY won't match. What do you suggest ? Also, according to DIPY's blog I'm one blog ahead.

**Comment by skoudoro** [comment]
let's meet to clarify all of that. it should be a quick meeting. we can do that on wednesday.

File: doc/posts/2023/2023_07_06_shilpi_week5.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Design Matrix Implementation and Coding with PEP8
+=================================================
+
```
Comment (skoudoro) [inline doc/posts/2023/2023_07_06_shilpi_week5.rst:3]: the header is missing 
```
.. post:: July 06 2023
    :author: Shilpi Prasad
    :tags: google
    :category: gsoc
```

File: doc/posts/2023/2023_07_06_shilpi_week5.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+What I did this Week
+~~~~~~~~~~~~~~~~~~~~
+
+This week I worked on improving the design_matrix that I'd previously created. It wasn't according to the PEP8 standards, I managed to modify it according to the appropriate format.
```
Comment (skoudoro) [inline doc/posts/2023/2023_07_06_shilpi_week5.rst:7]: design matrix: 2 words

File: doc/posts/2023/2023_07_06_shilpi_week5.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Design Matrix Implementation and Coding with PEP8
+=================================================
+
+What I did this Week
+~~~~~~~~~~~~~~~~~~~~
+
+This week I worked on improving the design_matrix that I'd previously created. It wasn't according to the PEP8 standards, I managed to modify it according to the appropriate format. 
+I mostly spent a bunch of time implementing functions under the Fit class in CTI. In order to achieve this task I had to look into functions which had already been implemented in DKI and QTI, as CTI is an extension of DKI and has similarities with the QTI model. 
+My procedure for this part was going through the 2 different modules and comparing the functions, and making note on functions which would have to be implemented in CTI as CTI has a very different design_matrix.
```
Comment (skoudoro) [inline doc/posts/2023/2023_07_06_shilpi_week5.rst:9]: strange sentence, to improve
design matrix: 2 words

File: doc/posts/2023/2023_07_06_shilpi_week5.rst [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+This week I worked on improving the design_matrix that I'd previously created. It wasn't according to the PEP8 standards, I managed to modify it according to the appropriate format. 
+I mostly spent a bunch of time implementing functions under the Fit class in CTI. In order to achieve this task I had to look into functions which had already been implemented in DKI and QTI, as CTI is an extension of DKI and has similarities with the QTI model. 
+My procedure for this part was going through the 2 different modules and comparing the functions, and making note on functions which would have to be implemented in CTI as CTI has a very different design_matrix. 
+Although this implementation is not completely done, I learnt a lot. 
+
+What is coming up next Week
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This week I intend to further implement the Fit class and also generate tests for the already implemented functions under the Fit class.
```
Comment (skoudoro) [inline doc/posts/2023/2023_07_06_shilpi_week5.rst:15]: `method` instead of `functions`

File: doc/posts/2023/2023_07_06_shilpi_week5.rst [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+This week I worked on improving the design_matrix that I'd previously created. It wasn't according to the PEP8 standards, I managed to modify it according to the appropriate format. 
+I mostly spent a bunch of time implementing functions under the Fit class in CTI. In order to achieve this task I had to look into functions which had already been implemented in DKI and QTI, as CTI is an extension of DKI and has similarities with the QTI model. 
+My procedure for this part was going through the 2 different modules and comparing the functions, and making note on functions which would have to be implemented in CTI as CTI has a very different design_matrix. 
+Although this implementation is not completely done, I learnt a lot. 
+
+What is coming up next Week
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This week I intend to further implement the Fit class and also generate tests for the already implemented functions under the Fit class. 
+And also write tests to make sure that the signals generated in the QTI model are the same as the ones done in CTI_pred.
```
Comment (skoudoro) [inline doc/posts/2023/2023_07_06_shilpi_week5.rst:16]: cti_pred : add backtick

**Review by skoudoro — CHANGES_REQUESTED** [review]
Hi @shilpiprd,

See below the comments. thanks for the future update

**Review by skoudoro — APPROVED** [review]
Thank you @shilpiprd for the update

it is much better. Merging

**Pull request merged**